crates/db_views/src/post_view.rs

```diff
@@ -358,6 +358,15 @@ fn queries<'a>() -> Queries<
           ));
         }
       }
+    } else {
+      match options.listing_type.unwrap_or_default() {
+        ListingType::Subscribed => {
```

Comment:
Subscribed doesn't make sense for this block, because `person_id` is None. It should be Local. Front ends should be hiding the subscribed button, but in case they don't you could make this filter work on `options.listing_type.unwrap_or_default() != ListingType::All`

Reply:
Right I mistyped this, fixed now.